app/controllers/claims/schools/claims_controller.rb

```diff
@@ -68,7 +68,10 @@ def remove; end
   def destroy
     @claim.destroy!
 
-    redirect_to claims_school_claims_path(@school, @claim), flash: { success: t(".success") }
+    redirect_to claims_school_claims_path(@school, @claim), flash: {
+      heading: t(".success"),
+      success: true,
```

Comment:
As Ben said (on Slack) – we don't need the explicit `success: true` because it defaults to showing a success banner. So let's drop it from all the controllers.